packages/zosfiles/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Added to the description of Create.ts `cmdType`. [#48](https://github.com/zowe/zowe-cli/issues/48)
```

Comment:
I suggest rewording the changelog text. We anticipate that our users will read the changlog to know if there are changes of interest to them. The reference to a source file is technically correct, but will not mean much to our users. Try stating the change in terms of the command that has been affected and the result that the end user will see. It still does not have to be very wordy - just a different perspective.

Reply:
Should this be considered a bugFix / patch since it's not introducing new behavior